ushahidi/platform-client
# refactoring loading of posts to avoid duplicate requests

This pull request makes the following changes:
- Refactors the way we load posts in the map to make sure we get the offset right and don't make duplicate calls.

Testing checklist:
This bug happened (sometimes) in deployments with > 200 posts with location
To test the fix:
Go to mapview in a deployment with clustering of markers enabled
- See posts loading
- Zoom out to full extent (only one marker with all posts)
- [ ] The count of posts should match the number of posts on the map (the same nb as is in the filter-dropdown and in mode-context bar).
- Open the filter-dropdown
- Add a filter
- Zoom out again if needed
- [ ] The count of posts should match the number of posts on the map (the same nb as is in the filter-dropdown and in mode-context bar).
 


Fixes ushahidi/platform#4064 .

Ping @ushahidi/platform
